src/browser/api/system.js

```diff
@@ -507,7 +490,7 @@ exports.System = {
     },
     setMinLogLevel: function(level) {
         try {
-            const mappedLevel = logLevelMappings.get(level + '');
+            const mappedLevel = log.logLevelMappings.get(level + '');
```

Comment:
Maybe leave a note that this won't accept inputs like `-1` or `3` but only strings like `'verbose'` or `'fatal'`

Reply:
I stared at this for too long until I looked at the javascript-adapter docs